rgw/v2/tests/s3_swift/test_object_expiration.py

```diff
@@ -0,0 +1,144 @@
+# test s3 bucket_lifecycle operations
```

Comment:
correct the comment, please.

Reply:
I have renamed the test_object_expiration.py file to test_bucket_lifecycle_object_expiration.py and similarly given the description as per that.